driver/vhci/vhci_ioctl.c

```diff
@@ -114,7 +114,6 @@ process_irp_urb_req(pusbip_vpdo_dev_t vpdo, PIRP irp, PURB urb)
 	DBGI(DBG_IOCTL, "process_irp_urb_req: function: %s\n", dbg_urbfunc(urb->UrbHeader.Function));
 
 	switch (urb->UrbHeader.Function) {
-	case URB_FUNCTION_GET_STATUS_FROM_DEVICE:
 	case URB_FUNCTION_GET_STATUS_FROM_INTERFACE:
 	case URB_FUNCTION_GET_STATUS_FROM_ENDPOINT:
 	case URB_FUNCTION_GET_STATUS_FROM_OTHER:
```

Comment:
How about applying the same approach to interface, endpoint or other?

Reply:
It may very well be a better option indeed, but I was not sure what needed to be implemented for these methods, and if the same code could be mostly reused..

Reply:
I've had a look and can't find a USB device that uses these, so I can't check if a potential implementation would work. Would you know of a way to trigger these URBs?

Reply:
I agree that it's not easy to check. Just implement and test later if some function driver will use. Or we try to test via winusb or libusb. But I feel it's a overkill. 

My comment intended that these cases are very similar to URB_FUNCTION_GET_STATUS_FROM_DEVICE and thus treating only DEVICE case looks  weird to me.

Reply:
You are right, I've checked some documentations around and implementing that should not be too hard, so I'll modify my commits. I'll also squash them into a single one for you to review.